src/EvfSessions.js

```diff
@@ -0,0 +1,52 @@
+import React, {useState, useMemo, useCallback} from 'react'
+import Sessions from '@keg-hub/tap-evf-sessions'
+import testData from './mocks/testData'
+import { evfModalBuilder } from './mocks/evfModalBuilder'
+import { useRGA4 } from '@keg-hub/rga4'
+
+const onBookingRequest = (sessionId, ids) => {
+  console.log('Booking request for session ', sessionId, 'with attendees: ', ids)
+  return new Promise(resolve => setTimeout(resolve, 1000))
+}
+
+const onWaitingRequest = (sessionId, ids) => {
+  console.log('Booking request for session ', sessionId, 'with attendees: ', ids)
+  return new Promise(resolve => setTimeout(resolve, 1000))
+}
+
+export const EvfSessions = props => {
+  const rga4 = useRGA4()
+
+  const [modalParentProps] = useState({ className: 'evf-modal' })
+
+  const SessionsModal = useMemo(() => {
+    return evfModalBuilder(modalParentProps)
+  }, [modalParentProps])
+
+  const bookingRequest = useCallback((sessionId, ids) => {
```

Comment:
events work as expected.
the only thing i noticed is that on the `booking_request` event, it fired twice consistently.
while the `waiting_request` is correctly firing once
- [image](https://user-images.githubusercontent.com/3317835/103576299-2badce00-4e90-11eb-8999-2e5b827862aa.png)

the code is exactly the same as the waiting list so I'm not sure why it would fire twice.

it doesn't seem like an issue on the `rga4` module and it could just be a bug with the analytics debug mode?

Reply:
I encountered this too

Reply:
Left comments about this [here](https://github.com/simpleviewinc/keg-test-consumer/pull/13#issuecomment-754244243)